src/snowflake/connector/cursor.py

```diff
@@ -849,6 +849,7 @@ def fetch_pandas_batches(self, **kwargs):
         self.check_can_use_pandas()
         if self._query_result_format != "arrow":  # TODO: or pandas isn't imported
             raise NotSupportedError
+        self._log_telemetry_job_data(TelemetryField.PANDAS_FETCH_BATCHES, 1)
```

Comment:
What does the 1 refer to in this case? Is it a count?

Reply:
I used this before, I think I'm to blame for doing this originally.
The reason is that `_log_telemetry_job_data` does `int(value)`, so instead of doing `True` and expecting `True` in the telemetry data, I explicitly did the conversion, as
```pycon
>>> int(True)
1
```

Reply:
Can we turn this into a constant that represents a True value in telemetry

Reply:
Mark explained it. We just need occurrence of the function call for now, so a dummy '1' is fine.

Reply:
We shouldn't have magic numbers. Someone down the line will have the same question that I did, and if we have a named constant its obvious what that 1 represents

Reply:
I made this parameter optional so when there is no value to provide, callers do not have to explicitly provide dummy value.